worker-batch-ingestion-plugin/src/main/java/com/github/cafdataprocessing/worker/ingestion/IngestionBatchWorkerPlugin.java

```diff
@@ -166,10 +168,7 @@ private void handleSubbatch(final String subbatch, final BatchWorkerServices bat
         log.debug("I am going to read each line of: " + subbatchFileName);
         final List<String> lines = new ArrayList<>();
         try {
-            if (!Files.exists(subbatchFileName)) {
-                log.error("Exception while reading subbatch: " + subbatch + ", it does not exist");
```

Comment:
Same comment on this log statement being discarded also.  (I would expect that the framework logs the exceptions so I would expect the removal of these log lines to be fine.)

Reply:
Did not see the thrown exceptions being logged in BatchWorker or StreamingWorkerWrapper. So added those log statements back.